### PR TITLE
fix: external url injection problem

### LINKED ISSUE #887: [Bug] External URL injection issues, such as side label icon
### Version

_No response_

### Current Behavior

https://github.com/user-attachments/assets/5bb51d7b-5108-4b63-a797-5db889305531

### Expected Behavior

_No response_

### Environment

```markdown
- OS:
- Browser:
```

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by Xsy41**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- [ ] build
- [ ] ci
- [ ] docs
- [ ] feat
- [x] fix
- [ ] perf
- [ ] refactor
- [ ] test

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

- resolve #887 

## Details
Convert openDiggerLogo and osGraphLogo to Base64 encoding.

image
image

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

File: src/pages/ContentScripts/features/repo-networks/base64.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...IYiwo1hjxzrhJ+YqyKKbY06yNDna3ndscmu/qNoOrrqgyewGFJU0AN9WXOVUZPIDdMk1GCAYTY2rSZQQ1xVpYm9+sMwie2jlWGj6cbHjjUU332aarK9IGsm/Xdo5MpH27WVJcq8wfWHWeEdLyUDx6ztUd4qS3EfVcXpAlqsjW4ZAMAJ0ej1HDWKF2NrXXEl3hGM7ymkJKkhqqpXS/MiKvHHCfpo3QM9UqN1UQCungI2GqB+v7FTlSr3gAHkSeB6ueC6wgxNMuyHweo7dnqQ2Coa66FBRZnabBzvSc+JAt7RXNSxzsJ1OsJ7dBQ9TdA2xveRAtUWqjmaow8OWNbgcoTqvJIuLPoUyhQz8/UIZX9KxeollFM/QTlRnF/KhoqFIi5rX4XcLfKo4/ESDVjOUA9kwSwjPep7FwxB6hj4hkldqfvOEC1Ced+r57EU9lDvajL4Val8XHJOnRHFk5QR2pZ6M16akjAJnTSglLOcue8arRFB5NuaZe7xXCDahLSKP1vk9NbaYm8yJodbdIp2JQtd2xOakLYb3eQeVoR3HbPDWqfuzofAMdehK3vVtSi0I8PVyN0aOlJoZZ4lgbidGyX3JFALfnUG8KemKyPkn4UfD4VAayhgLo8Cj9JHuH0+D8LJwqo0vPhb4yNBbbNoTlWqt1ZYnUb7XLpeNYnBjRm2c/k1MauF6AfUtXUY2pqvQkxmk6qlFe/ycRY5AuQ6pT3qan0+20QC/Fzyhkl6cz9heSHIvfbERSeW2SYFD3055RLbz63XbQIMEciWUzNQfnQE100lNO9NG+1xdTCs4b4WkzfG6Qo75uz8D/U7HS/zovqHA2Qe7+aJQB9ZNQ3Py6gj9JffYQp3crYOjNkZHxPqs/HNDie47bwO+n7U8F994leuznH6qSWzU3HDOLzlTjYxHwXOd5Jrh48bz0esLJc2UFFko1cnZ0Kc/xZQmUEtszzqRmDJev9kDVUQrCkzTyyh0pQHSJvu1MEVAQF32G3NIGavj6IhZymKKiRdUJsmk4WBzWSVmp9MDotUCrUJnB8VZzGVyTUwPGVXcv1YqEC2DnAarcRqLyg+iYmeo7jKhqqB1jL+TRA8VAdsObk4ml+MG9PIFA/w9WgA1qc7cpLatKmMA40qGK8TV6CQBUTqGICVUygiglUgSqWsP0CEkgZNgQoviEAAAAASUVORK5CYII=';
```
Comment (frank-zsy) [inline src/pages/ContentScripts/features/repo-networks/base64.ts:2]: How large is this image? We can convert to smaller image with less pixels to reduce the size of this string.

Comment (Xsy41) [reply src/pages/ContentScripts/features/repo-networks/base64.ts:2]: Yeah, I hadn't thought about it before. The original image was 117x117 px, I have now converted it to 36x36 px. This should reduce the size of the base64 string accordingly.

**Comment by tyn1998** [comment]
FYI, there is another way to use local assets directly to avoid the introduce of base64 images :) Maybe have a try

image

image

**Comment by frank-zsy** [comment]
If we can directly pack the image into the extension and use them in local env, that should be better than base64 encoding. Because usually base64 encoding will be a little bit larger than image format.

**Comment by Xsy41** [comment]
> FYI, there is another way to use local assets directly to avoid the introduce of base64 images :) Maybe have a try
> 
> image

I will follow this approach and directly pack the images into the extension. This will ensure better performance and avoid the overhead of using base64 encoding.

**Comment by wangyantong2000** [comment]
截屏2024-10-18 20 41 55

Finally, it is necessary to align the style with the original GitHub style. At present, when on the same line, it will be slightly higher than the original label.

**Comment by Xsy41** [comment]
Yes, this is the revised picture.

image

**Pull request merged**